### PR TITLE
Adding information/links to README.md

### PR DESCRIPTION
**Pull request by terencet-dev**
Updating README.md to reflect Apache 2.0 license.

### CODE REVIEW

**Review by SteveLasker — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -4,11 +4,36 @@ Notation is a project to add signatures as standard items in the registry ecosys
...
+## Core Documents
+
+* [Governance for Notation](https://github.com/notaryproject/notary/blob/master/GOVERNANCE.md)
+* [Maintainers and reviewers list](https://github.com/notaryproject/notary/blob/master/MAINTAINERS)
+
+## Community
+
+* Regular conversations for Notation occur on the [Cloud Native Computing Slack](https://app.slack.com/client/T08PSQ7BQ/CQUH8U287?) channel.
+
+* Weekly meetings occur each Monday. Please see the [CNCF Calendar](https://www.cncf.io/community/calendar/) for details.
```
Comment (SteveLasker) [inline README.md:29]: Can we avoid stating a date, and just point to the [CNCF calendar](https://www.cncf.io/calendar/) so we don't have to track changes in multiple locations?

**Review by SteveLasker — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (terencet-dev) [reply README.md]: Sounds good!

**Review by SteveLasker — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -4,11 +4,36 @@ Notation is a project to add signatures as standard items in the registry ecosys
...
 [prototype-2]:      https://github.com/notaryproject/notation/tree/prototype-2
 
+## Core Documents
+
+* [Governance for Notation](https://github.com/notaryproject/notary/blob/master/GOVERNANCE.md)
+* [Maintainers and reviewers list](https://github.com/notaryproject/notary/blob/master/MAINTAINERS)
+
+## Community
+
+* Regular conversations for Notation occur on the [Cloud Native Computing Slack](https://app.slack.com/client/T08PSQ7BQ/CQUH8U287?) channel.
```
Comment (bridgetkromhout) [inline README.md:27]: Do we want to name the channel?

File: README.md [hunk 3]
```diff
@@ -4,11 +4,36 @@ Notation is a project to add signatures as standard items in the registry ecosys
...
+
+* Please see the [CNCF Calendar](https://www.cncf.io/calendar/) for details.
```
Comment (bridgetkromhout) [inline README.md:29]: for community meeting details?

File: README.md (hunk 2 above)
Comment (terencet-dev) [reply README.md:27]: Like do you want me to replace the URL with this -> https://slack.cncf.io/ ?

File: README.md (hunk 3 above)
Comment (terencet-dev) [reply README.md:29]: can add that change

File: README.md (hunk 2 above)
Comment (terencet-dev) [reply README.md:27]: Oh, do you mean like:

Regular conversations for Notation occur on the  [CNCF - #notary-v2 Slack channel](https://app.slack.com/client/T08PSQ7BQ/CQUH8U287?)

**Review by SteveLasker — APPROVED** [review]
LGTM

**Pull request merged**